tutorial/PythOnBoardingBot/app.py

```diff
@@ -51,10 +51,10 @@ def onboarding_message(payload):
     event = payload.get("event", {})
 
     # Get the id of the Slack user associated with the incoming event
-    user_id = event.get("user", {}).get("id")
```

Comment:
For `team_join` events, this is correct. Could you revert this change?

Reply:
thx. reverted. This doesn't seem to be documented anywhere, except [here](https://api.slack.com/events/team_join) showing user is an object. Or am I missing something?

Reply:
Apart from the document you mentioned, there is no other so far.